static/examples/hashing-tool/index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>UID2 Hashing Tool</title>
+    <link rel="stylesheet" type="text/css" href="./app.css" />
+    <link rel="shortcut icon" href="/img/favicon.ico" />
+    <!-- <script src="https://cdn.integ.uidapi.com/uid2-sdk-3.2.0.js"></script> -->
+    <script src="./uid2-sdk-3.3.0.js"></script>
+    <!-- <script type="module">
+      //import * as exportsTest from './uid2-sdk-3.3.0.js';
+      //console.log(exportsTest);
+      //import { hashAndEncodeIdentifier } from '@uid2/uid2-sdk';
+       //import { hashAndEncodeIdentifier } from './uid2-sdk-3.3.0.js'
+      
+    </script> -->
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+  </head>
+  <script>
+    console.log('Initializing example.');
+    
+
+    
+
+    async function updateGuiElements(input) {
+      console.log('Updating displayed values.');
+      const input_type = document.querySelector('input[name="toggle_input_type"]:checked').value;
+      let normalizedInput = input;
+      if (input_type === "email") {
+        normalizedInput = window.__uid2Helper.normalizeEmail(input);
+        $('#normalization').text(normalizedInput);
+      }
+      
+      if (normalizedInput) {
+        const hashedValue = await window.__uid2Helper.hashIdentifier(normalizedInput);
+        $('#hashed').text(hashedValue);
+        const encodedValue = await window.__uid2Helper.hashAndEncodeIdentifier(normalizedInput);
+        $('#base64_encoded').text(encodedValue);  
+      }
+      else {
+        $('#normalization').text(' ');
+        $('#hashed').text(' ');
+        $('#base64_encoded').text(' ');
+        $('#alert_box_email').show();
+      }
+    }
+
+    function clearElements() {
+      document.querySelector('.input_value').value = "";
+      $('#normalization').text(' ');
+      $('#hashed').text(' ');
+      $('#base64_encoded').text(' ');
+      $('.alert').hide();
+    }
+
+    function onDocumentReady() {
+        console.log('Setting up interface handlers.');
+        document.querySelector('.input_value').placeholder = "Enter an email address";
+        $('#enter').click(handleEnter);
+        $('#clear_values_form').click(clearElements);
+        $('.alert').hide();
+        $('#clear_values_form').show();
+        $('#normalization').text(' ');
+        $('#hashed').text(' ');
+        $('#base64_encoded').text(" ");
+    }
+
+    function handleRadioClick(radioButton) {
+      clearElements();
+      const input_type = radioButton.value;
+      let placeholder = "";
+      if (input_type == "email") {
+        placeholder = "Enter an email address";
+        $('#normalization').show();
```

Comment:
nit.  since these fields always go together, you could put an id on a div containing both of them and just show/hide that.